packages/config/config/devices/0x0344/he-tps01.json

```
@@ -48,6 +48,11 @@
 			"$import": "templates/heltun_template.json#relay_output_mode",
 			"label": "Relay 1: Output Mode"
 		},
+		{
+			"#": "12",
+			"$import": "templates/heltun_template.json#relay_load_power",
+			"label": "Relay Load Power in watt"
```

```suggestion
			"label": "Relay 1: Load Power"
```
The unit is already part of the import.